jit/analyze.ts

```diff
@@ -220,6 +221,10 @@ module J2ME {
   }
 
   export function canYield(methodInfo: MethodInfo): YieldReason {
+    if (phase === ExecutionPhase.Runtime && methodInfo.codeAttribute && methodInfo.codeAttribute.code.length > 100) {
+      // Large methods are unlikely to not yield, so don't even bother checking at runtime.
+      return YieldReason.Unlikely;
```

Comment:
Nit: I'd reverse the logic here, calling this YieldReason.Likely, for large methods that are likely to yield, to avoid the double negative.